src/common/composables/amcharts5/index.ts

```diff
@@ -60,7 +62,7 @@ export const useAmcharts5 = (
         if (state.root) state.root.container.children.clear();
     };
 
-    const setChartColors = (chart: am5.SerialChart, colors: string[]) => {
```

Comment:
Is this type incorrect ?

Reply:
I deleted the treemap chart, not the serialchart category, it would be better to add a type. I'll fix it. Thanks.